### DISCUSSION TITLE
Unable to transfer ownership of organization

### DISCUSSION THREAD

**Discussion by jguelat** [opengisch/QField#3902] [Bugs]
I'm currently trying to transfer the ownership of an organization (I have admin rights) and I'm always getting an error (Server Error 500). The same happens if I try to delete the organization. Another colleague who is also admin of the same organization had the same problem.

Thanks in advance for your help!

**Comment by suricactus** [comment]
Will be fixed when https://github.com/opengisch/qfieldcloud/pull/516 is deployed with the next minor version.
